app/styles/components/clipboard.scss

```diff
@@ -1,33 +0,0 @@
-.copy-input-component {
-  @extend %clearfix;
-
-  .copy-icon {
-    cursor: pointer;
-    float: right;
-    margin: 9px 5px 0 0;
-
-    &.zeroclipboard-is-hover {
-      color: $color-dark-blue;
-    }
-  }
-
-  .form-control-wrapper {
-    margin-right: 40px;
-  }
-
-  .form-control {
-    background: #fff;
-    cursor: default;
-  }
-
-  .tooltip {
-    .tooltip-arrow {
-      border-bottom-color: #000;
-    }
-
-    .tooltip-inner {
-      background: #000;
-      color: #fff;
-    }
-  }
-}
```

Comment:
This is from an old clipboard bit that was replaced when `ember-cli-aptible-shared` was created.